trappy/ftrace.py

```diff
@@ -356,7 +356,8 @@ class definitions list.  Otherwise, register a class to parse
         for event_name in events:
             for cls in known_events.itervalues():
                 if (event_name == cls.unique_word) or \
-                   (event_name + ":" == cls.unique_word):
+                   (event_name + ":" == cls.unique_word) or \
+                   (event_name == cls.name):
```

Comment:
Personal opinion here - shouldn't we chose between **either** `unique_word` and `name` and stick with it ? This change means we can populate the `events` argument with both trace class names **and** trace event names, and we end up with dataframes named only after trace class names.

Moving to using trace class names as a "unified standard" will break existing code though, so this might not be the best thing to do. Your thoughts ?

Reply:
Hmm yeah I think you make a good point. The problem is that if you get it wrong and use the `name` instead of `unique_word`, the symptom is that you implicitly register a dynamic parser with a default implementation (given how long this took me to figure out, I think that would be completely undebuggable to someone who isn't quite intimate with TRAPpy). That's why I went for the 'accept both' solution here (where the symptom is impossible to debug) and the 'only use `unique_word` solution in the attribute name (where the symptom is an exception with a hopefully-useful message).

However, the `warnings` module lets us warn someone pretty loudly so maybe we can solve it that way... let me try it out.